migrations/frontend/1663569995_update_username_valid_chars_contraint/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Allow underscores in usernames.
+
+ALTER TABLE users
+	DROP CONSTRAINT users_username_valid_chars,
+    ADD CONSTRAINT users_username_valid_chars CHECK (username ~ '^\w(?:\w|[-.](?=\w))*-?$'::citext);
```

Comment:
```suggestion
ALTER TABLE users
	DROP CONSTRAINT users_username_valid_chars,
	ADD CONSTRAINT users_username_valid_chars CHECK (username ~ '^\w(?:\w|[-.](?=\w))*-?$'::citext);
```

Consistent use of tabs and spaces

Reply:
updated